tabbycat/users/forms.py

```diff
@@ -2,6 +2,7 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.forms import PasswordResetForm, SetPasswordForm, UserCreationForm, UsernameField
 from django.utils.translation import gettext_lazy as _
+from users.models import Membership # added 14/5/25 to allow for group membership in the InviteUserForm
```

Comment:
```suggestion

from .models import Membership
```

Don't need the comment, and can use a relative import here.